.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,7 @@
+contact_links:
+  - name: Get Enterprise Support
+    url: https://cilium.io/enterprise/
+    about: See Cilium Enterprise Distributions and Trainings
+  - name: Join Slack
+    url: https://cilium.io/enterprise/
```

Comment:
it's the same URL twice, is that intentional?

Reply:
No, I'm sorry 😞 fixed it now